alerta/plugins/blackout.py

```diff
@@ -19,9 +16,12 @@ class BlackoutHandler(PluginBase):
     any plugins for further notification.
     """
 
-    def pre_receive(self, alert):
+    def pre_receive(self, alert, **kwargs):
+        self.kwargs = kwargs
```

Comment:
Just realised, what will happen in case of multiple alerts (race condition)? 
Given we use one instantiation of plugin for all the alerts, it may cause wrong routing.

Reply:
One per thread, but yes, potentially a race condition. 😞

Reply:
I've gone with your original suggestion...
```
def get_config(self, key, default=None, type=None, **kwargs)
```
Usage ...
```
NOTIFICATION_BLACKOUT = self.get_config('NOTIFICATION_BLACKOUT', default=True, type=bool, **kwargs)
```
It's not quite as elegant as I wanted but in Python "explicit is better than implicit" so that's that.